vic/drivers/cesm/cpl_esmf/lnd_comp_esmf.F90

```diff
@@ -0,0 +1,178 @@
+
+    call ESMF_AttributeSet(export_state, name="lnd_prognostic", value=.false., rc=rc)
+    if (rc /= ESMF_SUCCESS) call ESMF_Finalize(rc=rc, endflag=ESMF_END_ABORT)
+
+    call ESMF_AttributeSet(export_state, name="rof_present", value=.false., rc=rc)
+    if (rc /= ESMF_SUCCESS) call ESMF_Finalize(rc=rc, endflag=ESMF_END_ABORT)
+
+    call ESMF_AttributeSet(export_state, name="sno_present", value=.false., rc=rc)
+    if (rc /= ESMF_SUCCESS) call ESMF_Finalize(rc=rc, endflag=ESMF_END_ABORT)
+
+    call ESMF_AttributeSet(export_state, name="sno_prognostic", value=.false., rc=rc)
+    if (rc /= ESMF_SUCCESS) call ESMF_Finalize(rc=rc, endflag=ESMF_END_ABORT)
+
+    convCIM  = "CIM"
+    purpComp = "Model Component Simulation Description"
+
+    call ESMF_AttributeAdd(comp,  &
+                           convention=convCIM, purpose=purpComp, rc=rc)
+
+    call ESMF_AttributeSet(comp, "ShortName", "VIC", &
+                           convention=convCIM, purpose=purpComp, rc=rc)
+    call ESMF_AttributeSet(comp, "LongName", &
+                           "Variable Infiltration Capacity Model", &
+                           convention=convCIM, purpose=purpComp, rc=rc)
+    call ESMF_AttributeSet(comp, "ReleaseDate", "2014", &
+                           convention=convCIM, purpose=purpComp, rc=rc)
+    call ESMF_AttributeSet(comp, "ModelType", "Land", &
+                           convention=convCIM, purpose=purpComp, rc=rc)
+
+!    call ESMF_AttributeSet(comp, "Name", "Sam Levis", &
+!                           convention=convCIM, purpose=purpComp, rc=rc)
+!    call ESMF_AttributeSet(comp, "EmailAddress", &
+!                           "slevis@ucar.edu", &
+!                           convention=convCIM, purpose=purpComp, rc=rc)
+!    call ESMF_AttributeSet(comp, "ResponsiblePartyRole", "contact", &
+!                           convention=convCIM, purpose=purpComp, rc=rc)
+
+end subroutine lnd_init_esmf
+
+!===============================================================================
+!BOP ===========================================================================
+!
+! !IROUTINE: lnd_run_esmf
+!
+! !DESCRIPTION:
+!     run method for the VIC lnd model
+!
+! !REVISION HISTORY:
+!
+! !INTERFACE: ------------------------------------------------------------------
+
+subroutine lnd_run_esmf(comp, import_state, export_state, EClock, rc)
+
+! !INPUT/OUTPUT PARAMETERS:
+   type(ESMF_GridComp)          :: comp
+   type(ESMF_State)             :: import_state
+   type(ESMF_State)             :: export_state
+   type(ESMF_Clock)             :: EClock
+   integer, intent(out)         :: rc
+
+
+!EOP
+
+   rc = ESMF_SUCCESS
+
+end subroutine lnd_run_esmf
+
+!===============================================================================
+!BOP ===========================================================================
+!
+! !IROUTINE: lnd_final_esmf
+!
+! !DESCRIPTION:
+!     finalize method for the VIC land model
+!
+! !REVISION HISTORY:
```

Comment:
I would remove all the lines:

```! !REVISION HISTORY:```

Since we are not planning to use this (we'll use github to track revision history)